mustang_codegen/lib/src/service_generator/screen_service_generator.dart

```diff
@@ -272,19 +244,6 @@ class ScreenServiceGenerator extends Generator {
                 'modelStr': jsonEncode(screenStateCache.toJson()),
               });
             }
-            if (t is Future) {
```

Comment:
this block of code is not getting invoked because mounted is always true as dispose method in the state never get called.

Reply:
okay